src/MoBi.UI/Views/EditNeighborhoodBuilderView.cs

```diff
@@ -0,0 +1,66 @@
+﻿using MoBi.Assets;
+using MoBi.Presentation.DTO;
+using MoBi.Presentation.Presenter;
+using MoBi.Presentation.Views;
+using OSPSuite.DataBinding;
+using OSPSuite.DataBinding.DevExpress;
+using OSPSuite.Presentation.Extensions;
+using OSPSuite.Presentation.Views;
+using OSPSuite.UI.Controls;
+using OSPSuite.UI.Extensions;
+
+namespace MoBi.UI.Views
+{
+   public partial class EditNeighborhoodBuilderView : BaseUserControl, IEditNeighborhoodBuilderView
+   {
+      private IEditNeighborhoodBuilderPresenter _presenter;
+      private readonly ScreenBinder<ObjectBaseDTO> _screenBinder = new ScreenBinder<ObjectBaseDTO>();
+
+      public EditNeighborhoodBuilderView()
+      {
+         InitializeComponent();
+      }
+
+      public void AttachPresenter(IEditNeighborhoodBuilderPresenter presenter)
+      {
+         _presenter = presenter;
+      }
+
+      public void Activate()
+      {
+         ActiveControl = tbName;
+      }
+
+      public void AddFirstNeighborView(IView view)
+      {
+         panelFirstNeighbor.FillWith(view);
```

Comment:
need to react to selection of those so that the view is enabled or disabled. Not sure how to do this yet
Can't remember how we are doing it normally